BlueprintUI/Sources/Environment/AdaptedEnvironment.swift

```diff
@@ -0,0 +1,41 @@
+import CoreGraphics
+
+/// Wraps an element tree with a modified environment.
+///
+/// By specifying environmental values with this element, all child elements nested
+/// will automatically inherit those values automatically. Values can be changed
+/// anywhere in a sub-tree by inserting another `AdaptedEnvironment` element.
+public struct AdaptedEnvironment: Element {
+    var wrappedElement: Element
```

Comment:
Nit: Just "wrapped"?

Reply:
The general swift convention is to leave the type name off of the var (eg people instead of peopleArray), but this is also fine.